guide/onboarding/automatic-cloud-backup.md

```diff
@@ -77,23 +79,23 @@ Below is an example of what the user interface might look like for an automatic
 
 </div>
 
-### How to discuss/explain It
+### How to discuss/explain it
 
-One approach is to explicitly explain within onboarding carousels that a user’s recovery phrase is stored on the cloud; this means that if they lose or break their device, they can regain access to their funds by logging into their respective cloud account on a different device.
+One approach is to explicitly explain within onboarding carousels that a user’s wallet backup is stored on the cloud; this means that if they lose or break their device, they can regain access to their funds on another device using their respective cloud account and Bitcoin wallet app.
 
 On a similar note, it’s crucial that you also explain that if someone else knows their cloud account information, they can gain access to their funds, too.
 
 {% include tip/open.html color="green" icon="check" label="Do: Inform your Users" %}
 
-Before automatically backing up a recovery phrase via cloud backup, make sure the user knows what they are about to do. Once the backup is complete, provide a summary and some reassurances before moving on. This is a good place for loading interactions and animation because there will likely be a few seconds between when a user clicks to upload to the cloud provider and when it completes.
+Before automatically creating a wallet backup via cloud backup, make sure the user knows what they are about to do. Once the backup is complete, provide a summary and some reassurances before moving on. This is a good place for loading interactions and animation because there will likely be a few seconds between when a user clicks to upload to the cloud provider and when it completes.
```

Comment:
```suggestion
Before creating a wallet backup via cloud backup, make sure the user knows what they are about to do. Once the backup is complete, provide a summary and some reassurances before moving on. This is a good place for loading interactions and animation because there will likely be a few seconds between when a user clicks to upload to the cloud provider and when it completes.
```